### PR TITLE
Allowdecline

### PR DESCRIPTION
**Pull request by quanlee**
Added AllowDecline to BaseSignatureRequest Class
Set AllowDecline to false in the BaseSignatureRequest constructor
Added allow_decline parameter with a value of 1 in the _PostSignatureRequest method if the AllowDecline is set to true

Tested from my staging server and it works.

### CODE REVIEW

File: HelloSign/HelloSign.cs [hunk 1]
```diff
@@ -460,6 +460,7 @@ private SignatureRequest _PostSignatureRequest(SignatureRequest signatureRequest
             if (signatureRequest.SigningRedirectUrl != null) request.AddParameter("signing_redirect_url", signatureRequest.SigningRedirectUrl);
             if (signatureRequest.UseTextTags) request.AddParameter("use_text_tags", "1");
             if (signatureRequest.HideTextTags) request.AddParameter("hide_text_tags", "1");
+            if (isEmbedded && signatureRequest.IsForEmbeddedSigning) request.AddParameter("is_for_embedded_signing", "1");
```
Comment (BHSPitMonkey) [inline HelloSign/HelloSign.cs]: Hi @quanlee - This actually isn't a parameter taken by these endpoints. `is_for_embedded_signing` only applies when creating an UnclaimedDraft (for use with [Embedded Requesting](https://www.hellosign.com/api/embeddedRequestingWalkthrough) flows). Can you revert this change?

**Comment by BHSPitMonkey** [comment]
@quanlee - Just pinging you again about the comment I left on the most recent commit. Cheers!

**Comment by quanlee** [comment]
Hi Stephen,

My apologies for the late response, I just saw this last night.  

Thanks for reminding me!  Great SDK btw, really saved me a lot of time.

Cheers,
Quan
